ports/oneTBB/common.mk

```
@@ -0,0 +1,96 @@
+ifndef QCONFIG
+QCONFIG=qconfig.mk
+endif
+include $(QCONFIG)
+include $(MKFILES_ROOT)/qmacros.mk
+
+NAME=oneTBB
+
+QNX_PROJECT_ROOT ?= $(PRODUCT_ROOT)/../../
```

```suggestion
QNX_PROJECT_ROOT ?= $(PRODUCT_ROOT)/../../$(NAME)
```
Removes redundant build command typing, `make -C ...` should directly work now. README.md and workflow file can be updated as well.